src/toolchain/export-android.md

```diff
@@ -0,0 +1,112 @@
+You need to compile your gdext library for each Android triple individually. Possible targets can be found by running:
+
+```bash
+rustup target list
+```
+
+You can find the linkers in the Android CLI directory at:
+
+```text
+{androidCliDirectory}/{androidCliVersion}/ndk/{ndkVersion}/toolchains/llvm/prebuilt/
+{hostMachineOs}/bin/{targetTriple}{androidVersion}
+```
+
+As of writing this, the tested triples are:
+
+| Triple                      | Environment Variable                            | Godot Arch     | GDExtension Config            |
+| --------------------------- | ----------------------------------------------- | -------------- | ----------------------------- |
+| `aarch64-linux-android`     | `CARGO_TARGET_AARCH64_LINUX_ANDROID_LINKER`     | `arm64`        | `android.debug.arm64`         |
+| `x86_64-linux-android`      | `CARGO_TARGET_X86_64_LINUX_ANDROID_LINKER`      | `x86_64`       | `android.debug.x86_64`        |
+| `armv7-linux-androideabi`   | `CARGO_TARGET_ARMV7_LINUX_ANDROID_LINKER`       | `arm32`        | `android.debug.armeabi-v7a`   |
+| `i686-linux-android`        | `CARGO_TARGET_I686_LINUX_ANDROID_LINKER`        | `x86_32`       | `android.debug.x86`           |
+
+Notice how the environment variables are in all-caps and the triple's "-" is replaced with "_".
+
+Make sure to add all of the triples you want to support to `rustup` via:
+
+```bash
+rustup target add {targetTriple}
+```
+
+Example:
+
+```bash
+rustup target add aarch64-linux-android
+```
+
+
+## A complete example
+
+Putting it all together, here is an example compiling for `aarch64-linux-android`. This is also probably the most common
+Android target, as of the writing of this.
+
+Assuming the following things:
+
+1. Android CLI is installed in the `$HOME` folder.
+2. Godot is still relying on Android NDK version 23.2.8568313. Check
+[here](https://github.com/godotengine/godot/blob/4a0160241fd0c1e874e297f6b08676cf0761e5e8/platform/android/java/app/config.gradle#L14).
+3. The downloaded Android CLI version is: 11076708_latest (update this to be the version you downloaded).
+4. This is being run on Linux. Change the `linux-x86_64` folder in `CLANG_PATH` and `CARGO_TARGET_AARCH64_LINUX_ANDROID_LINKER`
+to be your host machine's operating system.
+5. You are targeting Android version 34.
+
+And here is what the commands look like running from a bash shell:
+
+```bash
+rustup target add aarch64-linux-android
+
+export CLANG_PATH="$HOME/android-cli/11076708_latest/ndk/23.2.8568313/toolchains/llvm/prebuilt/linux-x86_64/bin/clang"
+export CARGO_TARGET_AARCH64_LINUX_ANDROID_LINKER=\
+"$HOME/android-cli/11076708_latest/ndk/23.2.8568313/toolchains/llvm/prebuilt/linux-x86_64/bin/aarch64-linux-android34-clang"
+
+cargo build --target=aarch64-linux-android
+```
+
+And then you should find a built version of your GDExtension library in:
+
+```text
+target/aarch64-linux-android/debug/{libraryName}.so
+```
```

Comment:
I just commented in the macOS/iOS PR that we should probably use `{YourCrate}` as the universal placeholder to refer to the user's crate, so it's consistent across pages.

Reply:
done :)